plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -1095,9 +1096,12 @@ fu_uefi_capsule_plugin_cleanup_esp(FuUefiCapsulePlugin *self, GError **error)
 		return FALSE;
 	esp_os_base = fu_uefi_get_esp_path_for_os(esp_path);
 	pattern = g_build_filename(esp_path, esp_os_base, "fw", "fwupd*.cap", NULL);
```

Comment:
I do wonder if we should just have `"*/fw/fwupd*.cap"` as the pattern...

Reply:
Yeah I think that sounds better